airbnb/src/components/Calendar/CalendarBody.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+import CalendarDates from './CalendarDates';
+
+function CalendarBody(props) {
+  const { totalDate } = props;
+
+  const lastDate = totalDate.indexOf(1);
+  const firstDate = totalDate.indexOf(1, 7);
+
+  return (
+    <Form>
+      {totalDate.map((date, idx) => {
+        return (
+          <CalendarDates
+            key={date}
+            idx={idx}
+            lastDate={lastDate}
+            firstDate={firstDate}
+            date={date}
+          ></CalendarDates>
+        );
+      })}
+    </Form>
+  );
+}
+const Form = styled.div`
```

Comment:
이름에 맞는 태그 사용하면 좋을 것 같습니다. 
```suggestion
const Form = styled.form`
```